spec/models/food_spec.rb

```diff
@@ -1,5 +1,42 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe Food, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { should validate_presence_of(:name) }
+  it { should validate_uniqueness_of(:name).scoped_to(:user_id) }
+  it { should validate_presence_of(:unit_name) }
+  it { should validate_presence_of(:user) }
+
+  it { should belong_to(:user) }
+  it { should have_many(:ingredients) }
+  it { should have_many(:recipes).through(:ingredients) }
+
+  describe ".aisle_options" do
+    subject { Food.aisle_options }
+    it "should be an array" do
+      expect(subject).to be_kind_of(Array)
+    end
+    it "should be made up of strings" do
+      expect(subject.map{ |item| item.class }.uniq).to eq([String])
```

Comment:
Space missing to the left of {.<br>Pass `&:class` as an argument to `map` instead of a block.